bundle/manifests/vault-config-operator.clusterserviceversion.yaml

```diff
@@ -661,6 +853,66 @@ spec:
     targetPort: 9443
     type: MutatingAdmissionWebhook
     webhookPath: /mutate-redhatcop-redhat-io-v1alpha1-randomsecret
+  - admissionReviewVersions:
```

Comment:
not sure what is going on here, but these will be added during the bundle build and should not be here

Reply:
Removed.